src/gatling/scala/uk/gov/hmcts/reform/cmc/performance/legalprocesses/LoginPage.scala

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.cmc.performance.legalprocesses
+
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ChainBuilder
+import io.gatling.http.Predef._
+import uk.gov.hmcts.reform.cmc.performance.simulations.checks.{CsrfCheck, CurrentPageCheck}
+import uk.gov.hmcts.reform.idam.LegalUser
+
+object LoginPage {
+
+  def logLegalIn(user: LegalUser)(implicit postHeaders: Map[String, String]): ChainBuilder = {
+    exec(http("Trigger login")
```

Comment:
this won't work on dev, login should be the same code between the two applications, it is just the different entry point

Reply:
done